### PR TITLE
NEW HBT test case using new ID: 11

### PR DESCRIPTION
**Pull request by harinineelu**

### CODE REVIEW

File: src/harness/testcases/WINNF_FT_S_HBT_testcase.py [hunk 1]
```diff
@@ -794,4 +794,34 @@ def test_WINNF_FT_S_HBT_9(self):
...
+
+    # Send a Heartbeat request.
+    heartbeat_request = [{
+        'cbsdId': cbsd_ids[0],
+        'grantId': grant_ids[0],
+        'operationState': 'AUTHORIZED'
+    }]
+    request = {'heartbeatRequest': heartbeat_request}
+    response = self._sas.Heartbeat(request)['heartbeatResponse'][0]
+    # Check the heartbeat response.
```
Comment (jni2000) [inline src/harness/testcases/WINNF_FT_S_HBT_testcase.py]: A minor comment. The test spec mentions to verify  that just one heartbeat response is included. Perhaps we can add a check to verify that no more than one response is present.

**Comment by harinineelu** [comment]
Thanks for the review. Added heartbeat response length check. Could you please take a look.

File: src/harness/testcases/WINNF_FT_S_HBT_testcase.py (hunk 1 above)
Comment (harinineelu) [reply src/harness/testcases/WINNF_FT_S_HBT_testcase.py]: Done

**Comment by harinineelu** [comment]
Merged #401

**Pull request merged**